src/store/traction/pacbio/runCreate/run.js

```diff
@@ -145,10 +187,13 @@ const newRunType = {
   label: 'Create Run',
 
   // returns the payload slightly different for new and existing runs
-  payload({ run, wells, smrtLinkVersion }) {
+  payload({ run, smrtLinkVersion }) {
     // eslint-disable-next-line no-unused-vars
     const { id, ...attributes } = run
-    return createPayload({ run: attributes, wells: Object.values(wells), smrtLinkVersion })
+    const plates = attributes.plates
+    delete attributes.plates
```

Comment:
Can the delete of attributes happen in `createRunPayload` for both create and update run